#!/usr/bin/env python
PACKAGE = "project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

integrationMethodEnum = gen.enum([gen.const("Euler", int_t, 0, "Euler integration"),
                                    gen.const("RungeKutta", int_t, 1, "RungeKutta integration")],
                                    "Enum of odometry integration methods")
gen.add("integrationMethod", int_t, 0, "Integration method parameter edited with an enum", 0, 0, 1, edit_method = integrationMethodEnum)

exit(gen.generate("project1", "odometry_publisher", "integrationParameter"))